### PR TITLE
Managing extension packages

### PR DESCRIPTION
**Pull request by BrainforgeUK**

### CODE REVIEW

File: administrator/components/com_plugins/models/forms/filter_plugins.xml [hunk 1]
```diff
@@ -36,6 +36,15 @@
 		>
 			<option value="">JOPTION_SELECT_ACCESS</option>
 		</field>
+
+		<field
+			name="package_id"
+			type="packagelist"
+			extensiontype="plugin"
+			onchange="this.form.submit();"
+		>
```
Comment (Quy) [inline administrator/components/com_plugins/models/forms/filter_plugins.xml]: Indent

Comment (BrainforgeUK) [reply administrator/components/com_plugins/models/forms/filter_plugins.xml]: Don't know what's the problem with the indentation. Looks consistent with the other fields.

Comment (brianteeman) [reply administrator/components/com_plugins/models/forms/filter_plugins.xml]: https://developer.joomla.org/coding-standards/xml.html

Comment (BrainforgeUK) [reply administrator/components/com_plugins/models/forms/filter_plugins.xml]: Thanks, changed.

**Comment by brianteeman** [comment]
i am really confused what is going on here

**Comment by BrainforgeUK** [comment]
I seem to have accidently triggered reviews of this pull request, messed up creating a pull request for another issue.
No changes since April - except the filter_plugins.xml indentation standards issue.

**Comment by brianteeman** [comment]
you have also changed the title and removed the desciptrion and have a lot of conflicting files

**Comment by BrainforgeUK** [comment]
Ideally want to revert / undo everything in this pull request after 11th Aprl 2018.
Or can it be deleted and start again with clean slate?
Been hanging around for a year!

**Comment by brianteeman** [comment]
I think you are getting confused. This pr was created just 23 hours ago

**Comment by brianteeman** [comment]
You have added completely unrelated code3 to this issue

You must use a different branch and create a different pull request for each issue you are trying to resolve

In your git client revert this latest change, commit and push

Then create a NEW branch and pull from staging
Make your new changes, commit and push
go to https://github.com/joomla/joomla-cms and you will see info that you can create a new pull request from your new branch

**Comment by brianteeman** [comment]
I am closing this - please submit a new pr

**Pull request closed without merging**